### PR TITLE
Mb 25 glowing effect

### PR DESCRIPTION
**Pull request by insertStringHere**
Not working as perfect as intended, but functions correctly, even in multiplayer.

There's a pretty significant graphical load for some reason; warrants more research, but for now this works.

### CODE REVIEW

**Review by solidwaterslayer — APPROVED** [review]
LGTM!

File: src/main/java/com/mineboundteam/minebound/event/GlowingEvent.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    @SubscribeEvent
+    public static void displayGlowing(RenderLivingEvent.Post<?,?> e) {
+        if(!isRendering && glowActive){
+            if(outlineBufferSource == null){
+                outlineBufferSource = new OutlineBufferSource(GlowingEvent.mc.renderBuffers().bufferSource());
+                outlineBufferSource.setColor(0xee, 0xee, 0xee, 255);
+            }
+
+            LivingEntity entity = e.getEntity();
+            if (!entity.equals(mc.player) && glowActive && entity.distanceTo(mc.player) < 200) {
```
Comment (MattKirchh) [inline src/main/java/com/mineboundteam/minebound/event/GlowingEvent.java:46]:
```suggestion
            if (!entity.equals(mc.player) && entity.distanceTo(mc.player) < 200) {
```
This is now redundant.

File: src/main/java/com/mineboundteam/minebound/event/GlowingEvent.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import com.mineboundteam.minebound.MineBound;
+import com.mineboundteam.minebound.registry.KeyRegistry;
+import com.mojang.blaze3d.vertex.PoseStack;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.renderer.OutlineBufferSource;
+import net.minecraft.client.renderer.RenderBuffers;
+import net.minecraft.client.renderer.entity.EntityRenderDispatcher;
+import net.minecraft.util.Mth;
+import net.minecraft.world.entity.Entity;
```
Comment (MattKirchh) [inline src/main/java/com/mineboundteam/minebound/event/GlowingEvent.java:14]: Imports on lines 7, 11, and 14 can also be removed as they are not used. 

The imports that were removed did not break anything.

Comment (insertStringHere) [reply src/main/java/com/mineboundteam/minebound/event/GlowingEvent.java:14]: EntityRenderDisplatcher and LivingEntity are used in the code, are you sure removing those two won't cause an issue?

File: src/main/java/com/mineboundteam/minebound/event/GlowingEvent.java (hunk 1 above)
Comment (insertStringHere) [reply src/main/java/com/mineboundteam/minebound/event/GlowingEvent.java]: I thought I removed it when moving the conditions lol

File: src/main/java/com/mineboundteam/minebound/event/GlowingEvent.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+import org.lwjgl.glfw.GLFW;
+
+import com.mineboundteam.minebound.MineBound;
+import com.mineboundteam.minebound.registry.KeyRegistry;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.renderer.OutlineBufferSource;
+import net.minecraft.client.renderer.entity.EntityRenderDispatcher;
+import net.minecraft.util.Mth;
+import net.minecraft.world.entity.LivingEnity;
```
Comment (MattKirchh) [inline src/main/java/com/mineboundteam/minebound/event/GlowingEvent.java:12]:
```suggestion
import net.minecraft.world.entity.LivingEntity;
```
This somehow got typod to LivingEnity.

**Pull request merged**